versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
 
 You can also set up multiple receivers by using the `continue` option for a route, so that the alerts sent to a receiver continue being evaluated in the next level of the routing tree, which could contain another receiver.
 
+## Configuring AlertmanagerConfig After A Chart Upgrade
+
+If you use SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change that helps prevent naming collisions.
+
+You must manually edit the configuration map to continue to receive alerts:
+
+1. Click on **☰** and select **local**.
```

Comment:
This should just be any cluster that has SMS/MS Teams alerts using the affected chart version and not specific to the local cluster (the description of the linked issue mentions downstream).